### PR TITLE
chore(build): merge jiva-csi and jiva-operator repos

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

This PR merges the jiva-operator and jiva-csi repos together. By the merging, the two repos the dependency management would be reduced and the tests can be run in a more effective way.

Github action build: https://github.com/shubham14bajpai/jiva-operator/actions/runs/514739252

Note: The travis file will be removed after the multi-arch builds are enabled from github actions

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,15 +55,6 @@ export XC_ARCH
...
 
-ifeq (${BASEIMAGE}, )
-ifeq ($(ARCH),linux_arm64)
-  BASEIMAGE:=arm64v8/ubuntu:18.04
-else
-	BASEIMAGE:=ubuntu:16.04
-endif
-endif
-export BASEIMAGE
-
```
Comment (utkarshmani1997) [inline Makefile:66]: Are we not going to support arm builds, or has it been handled elsewhere

**Review by utkarshmani1997 — APPROVED** [review]
@shubham14bajpai this LGTM, left a minor comment also can you add the ci tests in travis?

File: Makefile (hunk 1 above)
Comment (shubham14bajpai) [reply Makefile:66]: We will be building multi-arch builds using docker buildx github action. Will take that in separate PR.

**Review by prateekpandey14 — COMMENTED** [review]
bin files got added... which can be removed

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      run: make license-check
+
+    - name: verify dependencies
+      run: make deps
+
+    - name: verify tests
+      run: make test
+
+  e2e-tests:
+    # needs: ['unit-tests']
```
Comment (prateekpandey14) [inline .github/workflows/build.yaml:58]: check if these needs to be enabled

Comment (shubham14bajpai) [reply .github/workflows/build.yaml]: Done

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**